### DISCUSSION TITLE
Is there a way I can use markdown for links in SLACK_MESSAGE

### DISCUSSION THREAD

**Discussion by poponuts** [rtCamp/action-slack-notify#163] [General]
I tried this but doesn't work:
`SLACK_MESSAGE: ${{ job.status == 'success' && '✅ Tests passed' || '⛔ Tests failed - refer to this [guide](https://github.com/rtCamp/action-slack-notify/discussions/new?category=general) }}`

I tried including `SLACKIFY_MARKDOWN: true` but still doesn't work.

**Comment by poponuts** [comment]
Ignore this. The correct format is:
`SLACK_MESSAGE: ${{ job.status == 'success' && '✅ Tests passed' || '⛔ Tests failed - refer to this <(https://github.com/rtCamp/action-slack-notify/discussions/new?category=general|guide> }}`

**Comment by vkruglikov** [comment]
I just passed the envs

```
SLACKIFY_MARKDOWN: true
SLACK_MESSAGE: [here is an example](https://example.com)
```

**Comment by chriscarman141** [comment]
Can I help you with that send me your email to text you in google chat
